core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -214,14 +214,16 @@ THE SOFTWARE.
 
           <!-- add YUI logger if debugging YUI -->
           <j:if test="${h.yuiSuffix=='debug'}">
-          <!-- script to transform this into the Logger Console is done in hudson-behavior.js -->
-          <div id="yui-logreader" style="margin-top:1em"/>
+            <!-- script to transform this into the Logger Console is done in hudson-behavior.js -->
+            <div id="yui-logreader" style="margin-top:1em"/>
           </j:if>
         </div>
       </j:if>
       <div id="main-panel">
-         <j:set var="mode" value="main-panel" />
-         <d:invokeBody/>
+        <j:set var="mode" value="app-bar" />
```

Comment:
I wanted to move the app-bar (the title/controls row) to `layout.jelly` to make it easier to use in the future, so that devs wouldn't need to add all the `div` structuring for it. 

This way from a page all you need to do is:

```
<l:app-bar title="${%Plugin Manager}">
      <div class="jenkins-search">
        <input
          class="jenkins-search__input"
          type="search"
          id="filter-box"
          value="${request.getParameter('filter')}"
          placeholder="${filtered == 'true' ? '%Search' : '%Filter'}"/>
        <div class="jenkins-search__icon">
          <l:ionicon name="search-outline"/>
        </div>
      </div>
    </l:app-bar>
```

to use it. Hope that answers your question?